src/main/java/com/softserve/academy/spaced/repetition/domain/CourseRating.java

```diff
@@ -15,8 +15,9 @@
     @Column(name = "account_email", nullable = false)
     private String accountEmail;
 
-    @Column(name = "course_id", nullable = false)
-    private long courseId;
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "course_id")
```

Comment:
I suppose one course can be rated by many users. one-to-many, isn't it?

Reply:
But with OneToOne all is working.
Logically it must work with ManyToOne, but I don't understand how it works.

Reply:
Maybe it is like refence key but is interpreted by idea as ManyToOne?

Reply:
I think I should work with @ ManyToOne(fetch = FetchType.LAZY) and possibly @JsonIgnore. And no mapping from Course to CourseRating.

Reply:
Ok, I will try it.